src/components/NacelleReferences/ReferenceTable.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react'
+import {
+  Table,
+  Popover,
+  IconButton,
+  Box,
+  Button
+} from '@contentful/f36-components'
+import { Icon } from '@contentful/forma-36-react-components'
+import { css } from 'emotion'
+import arrowForward from '../../arrowForward.svg'
+
+export interface ReferenceTableRow {
+  type: string
+  field: string
+  nacelleReference: string
+  link: string
+}
+
+const ReferenceTable = (props: { data: ReferenceTableRow[] }) => {
+  const [openIndex, setOpenIndex] = useState(-1)
+
+  const row = {
+    border: '0',
+    margin: '0',
+    background: 'red'
+  }
+
+  const tableRows = props.data.map((item, index) => {
+    return (
+      <Table.Row key={index} className={css(row)}>
```

Comment:
Might be nice to avoid using the `index` as the `key`:

```suggestion
      <Table.Row key={`${item.field}::${item.type}`} className={css(row)}>
```

From [the React docs](https://react.dev/learn/rendering-lists#why-does-react-need-keys):

> You might be tempted to use an item’s index in the array as its key. In fact, that’s what React will use if you don’t specify a key at all. But the order in which you render items will change over time if an item is inserted, deleted, or if the array gets reordered. Index as a key often leads to subtle and confusing bugs.